packages/backend/server/src/plugins/copilot/context/service.ts

```diff
@@ -0,0 +1,69 @@
+import { Injectable } from '@nestjs/common';
+import { PrismaClient } from '@prisma/client';
+
+import { CopilotInvalidContext, CopilotSessionNotFound } from '../../../base';
+import { ContextSession } from './session';
+import { ContextConfig, ContextConfigSchema } from './types';
+
+@Injectable()
+export class CopilotContextService {
+  private readonly sessionCache = new Map<string, ContextSession>();
```

Comment:
How many sessions will exist simultaneously here? No cleanup mechanism? Why choose in-process memory caching?

Reply:
during the poc stage, this function was not injectable
replaced with redis cache now